src/App.jsx

```diff
@@ -27,11 +27,16 @@ function App() {
         setTodoList([...todoList, newTodo]);
     };
 
+    const removeTodo = (id) => {
+        const updateTodoList = todoList.filter((todo) => todo.id != id);
```

Comment:
It doesn't really matter here, but we should strive to use `===` and `!==` rather than `==` and `!=` unless you absolutely need to. There may be unexpected consequences in some cases of using == or != (loose comparison) rather than strict comparison, although I don't anticipate it will be a problem here.